src/helics/core/zmq/ZmqCommsSS.cpp

```diff
@@ -501,14 +501,14 @@ int ZmqCommsSS::processRxMessage (zmq::socket_t &brokerSocket,
     if (serverMode)
     {
         brokerSocket.recv (&msg1);
-        std::string str (static_cast<char *> (msg1.data ()), msg1.size ());
+        // std::string str (static_cast<char *> (msg1.data ()), msg1.size ());
```

Comment:
Is there a reason to keep lines 504 and 511 around?

Reply:
My guess is @shwethanidd  put them in for debugging, but we can probably delete them